.github/actions/spelling/expect.txt

```diff
@@ -9,6 +9,7 @@ AICLIC
 ajor
 amrutha
 anonymized
+ANSISTRING
```

Comment:
Probably from [microsoft/terminal](https://github.com/microsoft/terminal/blob/main/.github/actions/spelling/allow/apis.txt) where I specifically split out the different categories. But yeah, this would be better in `allow`.